SECURITY.md

```diff
@@ -0,0 +1,46 @@
+# Security policy
+
+## Supported versions
+<!-- Include start supported versions -->
+The release model of the core22 snap is following rolling releases. 
+The snap is released into the edge channel, and automatically promoted to 
+the beta channel if a snap revision is not already in progress for beta validation. 
```

Comment:
To be clear, there is an "edge validation" process that happens before promotion to beta, so some tests are performed to the edge snap too. Maybe we can also mention that additional tests involving real hw are performed in beta before promotion to candidate. And that we let the snap age in candidate for a while before promoting to stable, so candidate can get some external testing.

Reply:
Good point, synced with Sergio to understand the edge-validation better as well